extension/runtime/src/main/java/io/quarkiverse/temporal/WorkflowStub.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkiverse.temporal;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import jakarta.enterprise.util.Nonbinding;
+import jakarta.inject.Qualifier;
+
+@Qualifier
+@Retention(RUNTIME)
+@Target({ METHOD, FIELD, PARAMETER, TYPE })
+public @interface WorkflowStub {
```

Comment:
Temporal already have an interface named `io.temporal.client.WorkflowStub`, maybe we should rename this ?

Reply:
Yes we should rename this.  I am back to all of our custom Quarkus annotations should be `@TemporalXXX` this way we KNOW its ours and I know they didn't in Spring Boot but `@TemporalActivity` and `@TemporalWorkflowStub` are so much cleaner.

Reply:
done !